### PR TITLE
documentation: Fixed a typo in Rpi Pico index page

### LINKED ISSUE #67435: doc: Documentation issue in 'boards/arm/rpi_pico/doc/index'
**Describe the bug**

The CYW43439 is mentioned, it gives Wi-Fi and Bluetooth, but there is no explicit mention it isn't supported so the first-time visitor is left wondering. I'm left assuming it isn't supported given the following KConfig table.

Also typo: 'Bluetoth' which hinders search.

**Environment**

* Page: `boards/arm/rpi_pico/doc/index`
* Version: main
* SHA-1: cca5e3c0a46ad28bf8301c30999becb0157c02ac

### PR DESCRIPTION
**Pull request by HimaSava**
Fixes #67435

There was a typo on the index page of Rpi Pico. Changed "Bluetoth" to "Bluetooth".

### CODE REVIEW

**Review by soburi — APPROVED** [review]

File: boards/arm/rpi_pico/doc/index.rst [hunk 1]
```diff
@@ -9,7 +9,7 @@ Overview
 The Raspberry Pi Pico and Pico W are small, low-cost, versatile boards from
 Raspberry Pi. They are equipped with an RP2040 SoC, an on-board LED,
 a USB connector, and an SWD interface. The Pico W additionally contains an
-Infineon CYW43439 2.4 GHz Wi-Fi/Bluetoth module. The USB bootloader allows the
+Infineon CYW43439 2.4 GHz Wi-Fi/Bluetooth module. The USB bootloader allows the
```
Comment (soburi) [inline boards/arm/rpi_pico/doc/index.rst:12]: Please fix CI error.

**Review by soburi — CHANGES_REQUESTED** [review]

**Comment by soburi** [comment]
@HimaSava 
CI errors are still remaining.
https://github.com/zephyrproject-rtos/zephyr/actions/runs/7511136103/job/20454348524?pr=67566

**Review by henrikbrixandersen — APPROVED** [review]

**Comment by henrikbrixandersen** [comment]
@HimaSava You'll need to fix the commit log for this to be merged:

3: UC4 Commit message body line exceeds max length (82>75): "There was a typo on the index page of Rpi Pico. Changed "Bluetoth" to "Bluetooth"."
Error: See https://docs.zephyrproject.org/latest/contribute/guidelines.html#commit-guidelines for more details

 cd1893fd14b8e95d212ff7c0d6091f3b8631b780: author email (Himanshu Savargaonkar <H.Savargaonkar@student.tudelft.nl>) needs to match one of the signed-off-by entries.
Error: Process completed with exit code 1.

**Comment by soburi** [comment]
@HimaSava

Could you rebase it, please?

**Comment by nashif** [comment]
has been already addressed.

**Pull request closed without merging**